backend/src/db/repositories/PostRepository.ts

```diff
@@ -105,7 +106,7 @@ export default class PostRepository {
                 left join user_bookmarks b on (p.post_id = b.post_id and b.user_id=:user_id)
                 left join post_votes v on (v.post_id = p.post_id and v.voter_id=:user_id)
             order by
-                p.commented_at desc
+                p.${sortBy === FeedSorting.postCommentedAt ? 'commented_at' : 'created_at'} desc
```

Comment:
Please add index for `created_at desc`. Also I noticed that index for `commented_at` also missing 😱

Reply:
both `commented_at` and `created_at` are used in multi-field indexes ([commented_at](https://github.com/spaceshelter/orbitar/blob/42ebfd368935144ed943f5fa327a4cc1efb5bb21/mysql/schema.sql#L97), [created_at](https://github.com/spaceshelter/orbitar/blob/c668cb29684c654a898ef4b84dcae3284d09e29e/backend/migrations/20220511160411-posts-add-author-created-index.js#L18)), which should work when sorting. I could be wrong though, but when I did explain it showed these indexes as possible to use